docs/content/users/quickstart.md

```diff
@@ -767,12 +754,34 @@ If your project uses a database you'll want to set the [DB connection string](ht
     ```bash
     git clone <my-symfony-repo> my-symfony-site
     cd my-symfony-site
-    ddev config --docroot=public
+    ddev config --project-type=symfony --docroot=public
     ddev start
     ddev composer install
     ddev launch
     ```
 
+!!!tip "Default web environments"
+    On `ddev start` command, some environments will be automatically added to `.env.local` file.
+
+    - `MAILER_*` to use [Mailpit](https://ddev.readthedocs.io/en/stable/users/usage/developer-tools/#email-capture-and-review-mailpit) as mail catcher.
+    - `DATABASE_*` based on database selection unless [--omit-containers=db](https://ddev.readthedocs.io/en/stable/users/configuration/config/#omit_containers) is set.
+    - `REDIS_*` if `ddev/ddev-redis` or `ddev/ddev-redis-7` add-on installed.
```

Comment:
I don't think we need this "Default web environments" tip.

The place for this kind of info is here (where the general logic is described)
https://ddev.readthedocs.io/en/stable/users/usage/cms-settings/

Reply:
i thought that is just information those envs would be auto-populated and no user intervention was needed for that

Reply:
in the existing doc https://ddev.readthedocs.io/en/stable/users/quickstart/#symfony it state that the user need to modify the `.env` file with some string based on the configuration hence I do replace those statement with a tip that state user doesn't need to do anything

or we can copy from Laravel https://ddev.readthedocs.io/en/stable/users/quickstart/#laravel 
```
DDEV automatically updates or creates the .env file with the database information.
```

Reply:
I think we can have the same message as in Laravel quickstart (with `.env.local` instead of `.env`).

Reply:
Updated the docs with a short message instead.